pulsar-function-go/pf/instanceConf.go

```diff
@@ -72,8 +73,8 @@ func newInstanceConf() *instanceConf {
 				SubscriptionType: pb.SubscriptionType(cfg.SubscriptionType),
 				InputSpecs: map[string]*pb.ConsumerSpec{
 					cfg.SourceSpecTopic: {
-						SchemaType:     cfg.SourceSchemaType,
-						IsRegexPattern: cfg.IsRegexPatternSubscription,
+						SchemaType:     "",
```

Comment:
Why do you change the `SchemaType` to an empty string?

Reply:
Good catch, I might've overlooked after some testing.
I'll revert that.